crates/sequencer/src/state.rs

```diff
@@ -253,4 +312,75 @@ mod tests {
         // When
         state.get_compiled_class_hash(*ONE_CLASS_HASH).unwrap();
     }
+
+    #[test]
+    pub fn dump_and_load_state() {
+        let mut state = State::default();
+
+        // setting up entry for state.classes
+        let class_hash = ClassHash(stark_felt!("0x1"));
+        let contract_class = get_contract_class(include_str!(
+            "./test_data/contracts/compiled/universal_deployer.json"
+        ));
+        let compiled_class_hash = CompiledClassHash(stark_felt!("0x1"));
```

Comment:
can you use @greged93 's constants? I think he wrote a similar flow

Reply:
in the constant.rs file

Reply:
I think we can, I have almost all values in there, and we can add if missing.

Reply:
makes sense, done via [this](https://github.com/kkrt-labs/ef-tests/pull/159/commits/3c67a7a263c39f7e900dc289da5db5f2f365fb4a) commit.